EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/caching_in_the_shop/basketpreview_cache_and_user_specific_data.md

### Working with html fragments

> Each service implementing the SessionDataInterface can return HTML fragments. The keys used should use the id of an container inside your site. eZ Commerce will replace this container automatically using the prepared HTML.

Reviewer: ```suggestion
Each service implementing the `SessionDataInterface` can return HTML fragments. The keys should use the ID of a container inside your site. eZ Commerce will automatically replace this container by using the below HTML example.
```
